src/functions-and-arrays.js

```diff
@@ -1,41 +1,145 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  }
+  else if (number1 < number2) {
+    return number2;
+  }
+  else {
+    return number1;
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  let longestWord = ""
+  if (!arr.length) {
+    return null;
+  }
+  else if (arr.length === 1) {
+    return arr[0];
+  }
```

Comment:
Here you could have defined `longestWord` as the first element and avoid lines 24 and 25

` let longestWord = wordsArr[0];`